scene/realistic-lighting-and-shadows/src/main/java/com/esri/samples/realistic_lighting_and_shadows/RealisticLightingAndShadowsController.java

```diff
@@ -78,26 +73,32 @@ public void initialize() {
       // set atmosphere effect to realistic
       sceneView.setAtmosphereEffect(AtmosphereEffect.REALISTIC);
 
-      // set a new calendar and add a date and time
+      // set a calendar with a date and time
       calendar = new GregorianCalendar(2018, 7, 10, 12, 00, 0);
       calendar.setTimeZone(TimeZone.getTimeZone("PST"));
 
-      // set the time label on the control panel
+      // set the sun time to the calendar
       sceneView.setSunTime(calendar);
 
-      // tidy string to just return date and time (hours and minutes)
-      dateFormat = new SimpleDateFormat("EEE MMM dd HH:mm");
-      dateFormat.setTimeZone(TimeZone.getTimeZone("PST"));
-      String dateAndTimeTidied = dateFormat.format(calendar.getTime());
+      // format the string to just return the date and time (hours and minutes)
+      formatted = new SimpleDateFormat("EEE MMM dd HH:mm");
+      formatted.setTimeZone(TimeZone.getTimeZone("PST"));
+      String formattedDateAndTime = formatted.format(calendar.getTime());
```

Comment:
Sorry about the confusion here! We should still give the `SimpleDateFormat` variable a representative name:

```suggestion
      // format the string to just return the date and time (hours and minutes)
      dateFormat= new SimpleDateFormat("EEE MMM dd HH:mm");
      dateFormat.setTimeZone(TimeZone.getTimeZone("PST"));
      String formattedDateAndTime = formatted.format(calendar.getTime());
```